homeassistant/components/sensor/strings.json

```diff
@@ -247,7 +247,7 @@
       "name": "VOCs"
     },
     "volatile_organic_compounds_parts": {
-      "name": "[%key:component::sensor::entity_component::volatile_organic_compounds::name%]"
+      "name": "VOCs ratio"
```

Comment:
It is not clear to me why the device class provided name isn't correct? Doesn't that mean this device class shouldn't be used?

Reply:
There are two device classes that have the same name (`volatile organic compounds parts` and `volatile_organic_compounds`). When using device class selectors like the one used in the `scrape` integration and the one proposed for the `sql` integration (I haven't confirmed if there are other cases like this) the UI shows two device classes with the same name, not differentiating both , which can cause user confusion.

Before:
![image](https://github.com/home-assistant/core/assets/31328123/7ee17813-75cc-455c-a830-d447b32054a0)

After:
![image](https://github.com/home-assistant/core/assets/31328123/4ceece2d-46b3-48af-8cfc-2321f8892377)

Reply:
Makes sense! Thanks!